zebra-test/src/vectors/block.rs

```diff
@@ -220,6 +220,10 @@ lazy_static! {
             (1_116_000, BLOCK_TESTNET_1116000_BYTES.as_ref()),
             (1_116_001, BLOCK_TESTNET_1116001_BYTES.as_ref()),
             (1_326_100, BLOCK_TESTNET_1326100_BYTES.as_ref()),
+            // Nu5
+            (1_599_199, BLOCK_TESTNET_1599199_BYTES.as_ref()),
+            (1_599_200, BLOCK_TESTNET_1599200_BYTES.as_ref()),
```

Comment:
Can we move this comment?

Block `1_599_199` is the last Testnet Canopy block, but the comment makes it look like the first NU5 block.

```suggestion
            (1_599_199, BLOCK_TESTNET_1599199_BYTES.as_ref()),
            // Nu5
            (1_599_200, BLOCK_TESTNET_1599200_BYTES.as_ref()),
```